internal/provisioner/bundledeployment/bundledeployment.go

```diff
@@ -264,7 +269,14 @@ func (p *bundledeploymentProvisioner) reconcile(ctx context.Context, bd *rukpakv
 		return ctrl.Result{}, err
 	}
 
-	rel, state, err := p.getReleaseState(cl, bd, chrt, values)
+	post := &postrenderer{
+		labels: map[string]string{
+			util.CoreOwnerKindKey: "BundleDeployment",
```

Comment:
nit: we have a constant variable for the BundleDeployment Kind: https://github.com/operator-framework/rukpak/blob/main/api/v1alpha1/bundledeployment_types.go#L26.

Reply:
Thanks!  I'll update.